mmv1/templates/terraform/examples/firebase_apple_app_basic.tf.erb

```diff
@@ -0,0 +1,22 @@
+resource "google_project" "default" {
+	provider = google-beta
+
+	project_id = "tf-test%{random_suffix}"
+	name       = "tf-test%{random_suffix}"
+	org_id     = "<%= ctx[:test_env_vars]['org_id'] %>"
+}
+
+resource "google_firebase_project" "default" {
+	provider = google-beta
+	project  = google_project.default.project_id
+}
+
+resource "google_firebase_apple_app" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
I think we likely want to have 2 samples in general:
- a minimal sample that sets only the minimally required fields
- a full sample that sets all optional values with example values